testing/gnome-online-accounts/APKBUILD

```diff
@@ -1,40 +1,37 @@
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=gnome-online-accounts
-pkgver=3.26.0
-pkgrel=1
-pkgdesc="client for GNOME web services"
-url="https://www.gnome.org/"
+pkgver=3.32.0
+pkgrel=0
+pkgdesc="Single sign-on framework for GNOME"
+url="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
 # limited by webkit2gtk
 arch="x86 x86_64 ppc64le aarch64"
-license="GPL"
-makedepends="gnome-desktop-dev libxslt itstool libxml2-utils 
-	webkit2gtk-dev json-glib-dev rest-dev libsecret-dev 
+license="GPL-2.0-or-later"
+makedepends="gnome-desktop-dev libxslt itstool libxml2-utils
+	webkit2gtk-dev json-glib-dev rest-dev libsecret-dev
 	docbook-xsl docbook-xml"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-source="https://download.gnome.org/sources/$pkgname/${pkgver%.*}/$pkgname-$pkgver.tar.xz"
-builddir="$srcdir/$pkgname-$pkgver"
+subpackages="$pkgname-dev $pkgname-lang"
+source="https://download.gnome.org/sources/gnome-online-accounts/${pkgver%.*}/gnome-online-accounts-$pkgver.tar.xz"
 
 build() {
-	cd "$builddir"
```

Comment:
I think at present it is better to leave any `cd "$builddir"` lines, these make back ports and security fixes easier. Perhaps not as relevant in `testing/` but this is my understanding of the current approach.

Reply:
I've been told that these can be removed for all ports in testing& community because abuild has supported this behaviour for over 6 months now.